### PR TITLE
Monocle3 slides update

### PR DESCRIPTION
**Pull request by wee-snufkin**
Fixed typos, images, links

### CODE REVIEW

**Review by hexylena — DISMISSED** [review]

**Comment by nomadscientist** [comment]
Screenshot 2023-01-22 at 17 14 19
@wee-snufkin Hi! Can you see if you can debug this issue and also respond to Helena? Then we can merge :)

File: topics/single-cell/tutorials/scrna-case_monocle3-trajectories/slides.html [hunk 1]
```diff
@@ -90,17 +90,17 @@
...
 
 ???
-- snapshot data with cells in different stages of differentiaiton
+- snapshot data with cells in different stages of differentiation
 - multiple methods and how they work in biological context
 --
 
 - It would be quite difficult to analyse a sample every few seconds to see how the cells are changing. Therefore, we assume that snapshot data encompass all naïve, intermediate, and mature cell states with sufficient sampling coverage to allow the reconstruction of differentiation trajectories.
-[Sagar and Grün, 2020](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115822/)
+<br /> [Sagar and Grün, 2020](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115822/)
```
[the remark this answers is not in the document]
Comment (wee-snufkin) [reply topics/single-cell/tutorials/scrna-case_monocle3-trajectories/slides.html]: Apologies, lost your suggestion! The citations are fixed now!

**Comment by wee-snufkin** [comment]
@nomadscientist All fixed now (also citations that Helena mentioned)! The issue you referred to was because of the link to the RStudio tutorial that I thought would be published by the time the slides are updated

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Looks great! Thanks @wee-snufkin

**Pull request merged**